.github/workflows/alchemy-sync-from-upstream.yml

```diff
@@ -0,0 +1,100 @@
+name: Sync Panther Analysis from Upstream - Alchemy
+
+on:
+  schedule:
+    # 15:00Z, every Wednesday
+    - cron: "00 15 * * 3"
+  workflow_dispatch: # or on button click
+
+permissions: write-all
+
+jobs:
+  check_upstream:
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Checkout OMGWINNING Panter Analysis
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 #4.2.2
+        with: 
+          token: ${{ secrets.GIT_ENV_API_TOKEN }}
```

Comment:
You don't need token for this. The default workflow will have the privileges to download the local repo.